### PR TITLE
Junction track kit

### PR DESCRIPTION
**Pull request by liach**

### CODE REVIEW

File: assets/railcraft/lang/en_US.lang [hunk 1]
```diff
@@ -1232,6 +1232,8 @@ track_kit.railcraft.railcraft_gated.tips=Track with built-in Gate\n§o§c- Apply
 track_kit.railcraft.railcraft_gated_one_way.name=Gated One-Way
 track_kit.railcraft.railcraft_gated_one_way.tips=Track with built-in One-Way Gate\n§o§c- Apply Redstone to open -
 
+track_kit.railcraft.railcraft_junction.name=Junction
+
```
Comment (3TUSK) [inline assets/railcraft/lang/en_US.lang:1236]: `track_kit.railcraft.railcraft_junction.tips=?`

Comment (liach) [reply assets/railcraft/lang/en_US.lang:1236]: See my first commit message. In 1.7, it didn't have tips for the kit but only for the track type, so I opened this pull request for feedbacks.

**Comment by liach** [comment]
Close after discussion on Discord. @3tusk I recommend you to get on IRC or discord.

**Comment by 3TUSK** [comment]
Which server and which channel?@liach

**Comment by liach** [comment]
@3TUSK https://github.com/CovertJaguar/Railcraft/blob/mc-1.10.2/README.md#official-links

**Comment by CovertJaguar** [comment]
If you are curious, it was rejected because the TrackKit item won't be available in the final release. I've already committed the code to hide it and make it impossible to acquire.

**Comment by 3TUSK** [comment]
Fair enough - -

**Comment by CovertJaguar** [comment]
Upon closer inspection, this is probably still required for things like Walia.

**Comment by liach** [comment]
Why? :confused:

**Pull request merged**